templates/cloud/partners/index.html

```diff
@@ -155,10 +155,9 @@ <h2 class="muted-heading">A selection of our cloud partners</h2>
 
 {% block footer_extra %}
 <script>
-    YUI().use('event-base', function (Y) {
-        Y.on('domready', function () {
-            core.loadPartners('?programme__name=Certified%20Public%20Cloud&programme__name=Charm%20Partner%20Programme&programme__name=OpenStack&featured=true', '#cloud-partners');
-        });
-    });
+    partnerLogoClouds = [{
```

Comment:
This isn't working. I assume because `partnerLogoClouds` is only interpreted in `partners.js`, which isn't currently included in this page. `partners.js` will have to load *after* `partnerLogoClouds` is defined to work.

It's lucky I thought to look for this specifically, otherwise I'd have never caught it, as it didn't throw any kind of error.

Which actually brings me to a slightly different topic. I'd rather we followed the pattern that was previously used in this page. In the old code, `core.loadPartners` was called from here. This would mean that if `core.loadPartners` wasn't available, you would see an error - as you should.

This method of more passively declaring a `partnerLogoClouds` variable is much more prone to failing silently.

So I would prefer that we took the `partners.loadPartnerClouds();` function call out of `partners.js` and put it in the bottom of every page that needs it. This would also allow us to pass through the partner information directly. Something like:

``` html
{% block footer_extra %}
<script src="static/js/partners.js"></script>
<script>
partners.loadPartnerCloud(
  '#cloud-partners',
  '?programme__name=Certified%20Public%20Cloud&programme__name=Charm%20Partner%20Programme&programme__name=OpenStack&featured=true'
);
// And if you had more than one cloud, call it again:
partners.loadPartnerCloud(
  '#another-cloud',
  '?programme__name=Certified%20Public%20Cloud'
);
</script>
{% endblock %}
```